src/main.ts

```diff
@@ -5,7 +5,7 @@ import { useMirageJS } from './plugins/miragejs';
 import 'windi.css';
 import App from './App.vue';
 
-if (Number(import.meta.env.VITE_MIRAGE_ENABLED)) {
```

Comment:
But how do I do run a dev build without MirageJS now?

Reply:
EXCELLENT POINT. I'll add an environment variable to disabled MirageJS over having to explicitly enable it as the most likely development case I think would be with the stub/mock running, right? Or would you rather have it disabled by default?